### PR TITLE
Added initial landing page

### PR DESCRIPTION
**Pull request by TheHollidayInn**
Here is a quick mockup. I haven't ever used Fastify, and I'm not 100% on how the badge works, but I hope this is helpful.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  <body>
+    <section class="mui-container">
+      <div class="mui-panel">
+        <h1>Package Maintenance Team</h1>
+
+        <p>Working Group for discussion on how to help ensure baseline maintenance and ability to safely use key packages in the ecosystem with current Node.js versions.
+          You can find more about this initiative in the article: Call to Action: Accelerating Node.js Growth</p>
+
+        <div class="mui-textfield">
+          <input id='package-name' type="text" placeholder="fastify">
```
Comment (Eomm) [inline public/index.html]:
```suggestion
          <input id='package-name' type="text" placeholder="package name">
```

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        </div>
+
+        <div class="mui-textfield">
+          <textarea id='badge-markdown' placeholder=""></textarea>
```
Comment (Eomm) [inline public/index.html:58]:
```suggestion
          <textarea id='badge-markdown' placeholder="" disabled></textarea>
```

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      </div>
+    </section>
+
+    <script>
+      window.onload = () => {
+        document
+          .querySelector('#generate-badge-button')
+          .addEventListener('click', async () => {
+            const package = document.querySelector('#package-name').value;
+            const badge = `![package maintenance status](${window.location.href}/backing/${package})`
```
Comment (Eomm) [inline public/index.html]: This generate an extra slash for me:
`package maintenance status`

And the `/backing/` part should be a `<select>` with options:
- target
- response
- backing
- response-paid
- url

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    </style>
+    <script src="//cdn.muicss.com/mui-0.9.41/js/mui.min.js"></script>
+  </head>
+  <body>
+    <section class="mui-container">
+      <div class="mui-panel">
+        <h1>Package Maintenance Team</h1>
+
+        <p>Working Group for discussion on how to help ensure baseline maintenance and ability to safely use key packages in the ecosystem with current Node.js versions.
+          You can find more about this initiative in the article: Call to Action: Accelerating Node.js Growth</p>
```
Comment (Eomm) [inline public/index.html]:
```suggestion
          You can find more about this initiative in the article: <a href="https://medium.com/@nodejs/call-to-action-accelerating-node-js-growth-e4862bee2919" target="_blank">Call to Action: Accelerating Node.js Growth</a></p>
```

**Review by Eomm — COMMENTED** [review]
Great! Thank you very much for this nice page!
I have added some improvements so we can start spreading this badge ✌

**Comment by TheHollidayInn** [comment]
Updated! I had to not use the disabled attribute because I'm unable to copy the textarea.

**Review by Eomm — APPROVED** [review]
I think it is perfect!

**Pull request merged**